File: assignment1/QuoRem.cpp
#include<iostream>
using namespace std;
class RemQuo
{
int divisor, dividend, quotient, remainder;
public: void display()
{
cout<<"enter dividend";
cin>>divisor;
cout<<"enter divisor";
cin>>divisor;
quotient=dividend/divisor;
remainder=dividend%divisor;
cout<<"quotient="<<quotient;
cout<<"remainder="<<remainder;
}
};
int main()
{
    RemQuo d1;
    d1.display();
}

File: QuoRem.java
import java.util.Scanner;
class QuoRem
{
public static void main(String[] args)
{
int Quotient, remainder,divisor, dividend;
Scanner sc=new Scanner(System.in);
System.out.println("enter dividend");
dividend=sc.nextInt();
System.out.println("enter divisor");
divisor=sc.nextInt();
Quotient=dividend/divisor;
remainder=dividend%divisor;
System.out.println(Quotient);
System.out.println(remainder);
}
}